LEGO1/legoinputmanager.h

```diff
@@ -28,10 +28,26 @@ class LegoInputManager : public MxPresenter
 
   virtual MxResult Tickle() override; // vtable+0x8
 
-  undefined m_pad40[0x15c];
+  undefined m_pad40[0x48];
+
+  MxBool m_unk88;
+  MxU8 m_unk89;
```

Comment:
Are those `U8` addressed anywhere? If not, they might just be implicit compiler padding and we shouldn't specify them imo